CircuitBreaker/src/FortuneTeller/Fortune-Teller-UI/Startup.cs

```diff
@@ -42,7 +42,8 @@ public void ConfigureServices(IServiceCollection services)
             services.AddControllersWithViews();
 
             // Add Hystrix metrics stream to enable monitoring
-            services.AddHystrixMetricsStream(Configuration);
+            //  services.AddHystrixMetricsStream(Configuration);
```

Comment:
If this line is still relevant for using the old dashboard, should we add a comment here that indicates when to use which?

Reply:
Added comments to clarify usage